dotnetcore/NetstandardModuleSample/src/NetstandardModuleSample/module_dev_sample.json

```diff
@@ -0,0 +1,55 @@
+﻿{
```

Comment:
Considering that you will have simulated_ble in your sample, you might need to change the simplest sample to be something more specific, like "simple"